app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -164,7 +164,7 @@ def render_updated_templates
   end
 
   def provisioning_is_cloud?
-    params[:template_klass] == 'cloud' || %w[auth_key_pair_cloud availability_zone cloud_tenant ems_cloud host_aggregate orchestration_stack vm_cloud miq_request].include?(params[:controller])
+    params[:template_klass] == 'cloud' || %w[auth_key_pair_cloud availability_zone cloud_tenant ems_cloud host_aggregate orchestration_stack vm_cloud].include?(params[:controller])
```

Comment:
This is kind of gross (not your change, the code already here).  Is it possible to just set template_klass in all cases?  That seems much more straightforward but not sure where this is coming from.